### PR TITLE
WIP Add Blurb check to lint command

### PR DESCRIPTION
**Pull request by nywilken**
This change introduces a new check to the lint command that will validate that a Track's config has a blurb with at least 10 words. This check does not verify the quality of the said blurb.

### CODE REVIEW

File: cmd/lint_test.go [hunk 1]
```diff
@@ -22,52 +19,86 @@ var (
...
-	defer func() {
-		ui.Out = originalOut
-		ui.ErrOut = originalErrOut
-	}()
-
-	lintTests := []struct {
-		desc     string
-		path     string
-		expected bool
+//func TestLintTrack(t *testing.T) {
```
Comment (kytrinyx) [inline cmd/lint_test.go:22]: Should we be deleting rather than commenting out the test?

File: cmd/lint.go [hunk 2]
```diff
@@ -146,6 +151,16 @@ func lintTrack(path string) bool {
 	return hasErrors
 }
 
+func incompleteBlurb(t track.Track) []string {
+	var tracks []string
+	re := regexp.MustCompile(`(?im:\b[\w]{4,}\b)`)
+	words := re.FindAllString(t.Config.Blurb, -1)
+	if len(words) < 10 {
```
Comment (kytrinyx) [inline cmd/lint.go:158]: The error message says 30, this says 10 (and the PR says 8)... should we make a constant that both places can refer to?

File: cmd/lint_test.go [hunk 3]
```diff
@@ -363,9 +394,9 @@ func TestLockedCoreViolation(t *testing.T) {
 			assert.Equal(t, tt.expectedSlug, slugs[0], tt.desc)
 		}
 	}
-}
+} // }}}
```
Comment (kytrinyx) [inline cmd/lint_test.go:397]: What are these comments for?

File: fixtures/numbers/config.json [hunk 4]
```diff
@@ -3,6 +3,7 @@
   "language": "Numbers",
   "repository": "https://github.com/exercism/numbers",
   "active": true,
+   "blurb": "Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.",
```
Comment (kytrinyx) [inline fixtures/numbers/config.json:6]: Let's make this stupider... like "One two three four five..." up to however many words we have required.

**Review by kytrinyx — COMMENTED** [review]
Overall this looks great. I have a couple of small comments/questions.

File: cmd/lint_test.go (hunk 1 above)
Comment (nywilken) [reply cmd/lint_test.go:22]: Oh forgot to to uncomment after my testing 🤦‍♂️

File: cmd/lint_test.go (hunk 3 above)
Comment (nywilken) [reply cmd/lint_test.go:397]: Oh sorry my vim markers for code folding 🤪

File: fixtures/numbers/config.json (hunk 4 above)
Comment (nywilken) [reply fixtures/numbers/config.json:6]: Sounds good. Thanks for taking a look over this PR.

**Comment by nywilken** [comment]
Given its age I think it’s safe to close and revisit if there is still a need.

**Pull request closed without merging**